com.jgraph.drawio.desktop.json

```diff
@@ -30,7 +30,7 @@
         "install -Dm644 usr/share/icons/hicolor/0x0/apps/draw.io.png /app/share/icons/hicolor/512x512/apps/com.jgraph.drawio.png",
```

Comment:
the icons also need to be exported with the new name. all files that are exported need the app-id

Reply:
Done